packages/amplify-python-function-template-provider/amplify-plugin.json

```diff
@@ -13,6 +13,10 @@
       {
         "name": "Hello World",
         "value": "hello-world"
+      },
```

Comment:
python e2e tests will need an update to "know about" this template option in the list